mmv1/third_party/tgc_next/pkg/cai2hcl/converters/services/compute/compute_instance_helpers_tgc.go

```diff
@@ -0,0 +1,194 @@
+package compute
+
+import (
+	"strings"
+
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/cai2hcl/converters/utils"
+	"github.com/GoogleCloudPlatform/terraform-google-conversion/v6/pkg/tpgresource"
+
+	compute "google.golang.org/api/compute/v0.beta"
+)
+
```

Comment:
The functions in this files are copied from mmv1/third_party/tgc_next/pkg/cai2hcl/converters/services/compute/compute_instance_helpers.go and renamed.

Reply:
Is there any particular reason to rename the functions and add "tgc" at the end?

Reply:
I see, do we expect more changes like this coming in?